rad/src/types/array.rs

```diff
@@ -13,9 +14,9 @@ use crate::{
     types::{RadonType, RadonTypes},
 };
 
-pub const RADON_ARRAY_TYPE_NAME: &str = "RadonArray";
+const RADON_ARRAY_TYPE_NAME: &str = "RadonArray";
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Serialize)]
```

Comment:
I think this is unused, as the `RadonArray` serialization is implemented in `impl Serialize for RadonTypes`. Probably the same applies to all the other radon types.

```suggestion
#[derive(Clone, Debug, PartialEq)]
```